workflows/format-check.yml

```
@@ -0,0 +1,56 @@
+name: "Format Check"
+
+on:
+  workflow_call:
+    inputs:
+      directory:
+        description: "The directory on which JuliaFormatter needs to be run"
+        default: "."
+        required: false
+        type: string
+      julia-version:
+        description: "Julia version"
+        default: "1"
+        required: false
+        type: string
+      juliaformatter-version:
+        description: "Version of JuliaFormatter to use"
+        default: "1.0.51"
```

```suggestion
        default: "1.0.50"
```

there's a bug in that one 😅